internal/resources/aws/elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,115 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// ElasticBeanstalkEnvironment struct represents <TODO: cloud service short description>.
+//
+// <TODO: Add any important information about the resource and links to the
+// pricing pages or documentation that might be useful to developers in the future, e.g:>
+//
+// Resource information: https://aws.amazon.com/elasticbeanstalk/
+// Pricing information: https://aws.amazon.com/elasticbeanstalk/pricing/
+type ElasticBeanstalkEnvironment struct {
+	Address string
+	Region  string
+	Name    string
+
+	InstanceCount    *int64
+	InstanceType     string
+	RDSIncluded      bool
+	LoadBalancerType string
+	StreamLogs       bool
+
+	RootBlockDevice     *EBSVolume
+	CloudwatchLogGroup  *CloudwatchLogGroup
+	LoadBalancer        *LB
+	ElasticLoadBalancer *ELB
+	DBInstance          *DBInstance
+	LaunchConfiguration *LaunchConfiguration
+}
+
+// ElasticBeanstalkEnvironmentUsageSchema defines a list which represents the usage schema of ElasticBeanstalkEnvironment.
```

Comment:
Could we add some more meaningful docs here mentioning that all the usage schema comes from the sub resources.